### PR TITLE
fix(cadre): health server check for nil

### PR DESCRIPTION
**Pull request by Matovidlo**

### CODE REVIEW

File: cadre.go [hunk 1]
```diff
@@ -151,16 +151,21 @@ func (c *cadre) startGRPC() {
 		t := time.NewTicker(5 * time.Second)
 
 		for {
+			// Nothing to check
+			if c.grpcHealthService == nil {
```
Comment (dbmtv) [inline cadre.go:155]: Can be moved outside the closure - there's no point starting a goroutine only to return from it.

File: http/responses/main_test.go [hunk 2]
```diff
@@ -502,11 +503,11 @@ func TestInternalError(t *testing.T) {
 			res := tt.httpTestShit.recorder.Result()
 
 			if res.StatusCode != tt.wantStatus {
-				t.Errorf("Created response invalid status code. want %d, got %d", tt.wantStatus, res.StatusCode)
+				assert.Equal(t, tt.wantStatus, res.StatusCode)
```
Comment (dbmtv) [inline http/responses/main_test.go:506]: The whole if statement can be replaced with the assertion. However, other parts of the file remain unchanged. Perhaps leave it for some other PR where all occurrences will be replaced?

**Review by dbmtv — CHANGES_REQUESTED** [review]

File: cadre.go (hunk 1 above)
Comment (Matovidlo) [reply cadre.go:155]: changed and moved to separate function

Comment (Matovidlo) [reply cadre.go:155]: changed and moved to separate function

File: http/responses/main_test.go (hunk 2 above)
Comment (Matovidlo) [reply http/responses/main_test.go]: Removed

**Review by dbmtv — APPROVED** [review]

**Pull request merged**